config/standard/test/targets.spec.js

```diff
@@ -61,13 +61,13 @@ describe('Standard Configuration Targets', function() {
 
           const expectedTargets = [
             {
-              _id: 'adult-1-pnc-active',
+              _id: 'adult-1~pnc-active',
```

Comment:
There is surely a rationale behind this change, but wasn't immediately obvious to me.

Reply:
So the id is contact-id-event-id-schedule-id or sometimes it is report-id-contact-id-schedule-id-targetconfig-id ... Since it's common to hyphenate these ids, we end up with ids like `c-c-1-c-r-1-0-5`. Even when merged they can be ambiguous and cause unexpected collisions which result in tasks not appearing or targets getting double counted. It isn't a great fix - but I just wanted something that made it slightly easier to parse these IDs and break it into the component parts. 

I didn't realise so many tests assert on the exact state of the ID. I don't think that asserting on something that is so internal to the working of the framework is a healthy testing practice. I probably wouldn't have made the change know this.

Reply:
Ah, thanks for explaining. Even if it is not a great fix, seems better than it was! Thanks